c_models/makefiles/extra.mk

```diff
@@ -0,0 +1,148 @@
+ifndef NEURAL_MODELLING_DIRS
+    $(error NEURAL_MODELLING_DIRS is not set.  Please define NEURAL_MODELLING_DIRS (possibly by running "source setup" in the neural_modelling folder within the sPyNNaker source folder))
+endif
+
+# ----------------------------------------------------------------------
+# Compute the absolute path to the directory containing this file.
+#
+EXTRA_MAKEFILE_PATH := $(abspath $(lastword $(MAKEFILE_LIST)))
+
+# Note: neural_build.mk will use abspath and add a final slash character as
+# needed, so DIRs can be in .. format here
+
+# ----------------------------------------------------------------------
+# Make sure that APP_OUTPUT_DIR points to where you want the .aplx files to go.
+#
+APP_OUTPUT_DIR := $(dir $(EXTRA_MAKEFILE_PATH))../../python_models8/model_binaries/
+
+# ----------------------------------------------------------------------
+# For multiple source directories, see note below.
```

Comment:
if its multiple source directories, why dos it then just talk about 1 extra src dir, and how would i seperate multiple? ,? spaces? list?

Reply:
see note below